resources/reporters/currents-playwright/README.md

```
@@ -58,39 +58,39 @@ use: {
+  - Explicitly add Currents reporter to `playwright.config.ts`
+  - Run `playwright test` as usual
+  - Keep Currents configuration in `currents.config.ts`
+- `pwc-p` command-line executable for orchestration.
+  - Run `pwc-p run` to orchestrate tests across CI machines
+  - Run `pwc-p discover` when filtering the suite (`--grep`, `--last-failed`, `--project`, spec paths)
+  - Required to enable [#playwright-orchestration](../../../guides/ci-optimization/playwright-parallelization.md#playwright-orchestration "mention")
+  - See [playwright-orchestration.md](../../../guides/ci-optimization/playwright-orchestration.md "mention") and [pwc-p.md](pwc-p.md "mention")
 
 ### `pwc` command-line executable
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)